src/components/EditSciApplication.vue

```diff
@@ -442,7 +452,7 @@ import BasicCustomField from '@/components/util/BasicCustomField.vue';
 import NotFound from '@/components/NotFound.vue';
 
 export default {
-  name: 'CreateSciApplication',
+  name: 'EditSciApplication',
```

Comment:
This was confusing me because the filename is EditSciApplication.vue but the component name didn't line up, so update that